tf2_ros/include/tf2_ros/buffer_interface.h

```diff
@@ -69,6 +69,12 @@ namespace tf2_ros
     return (s + std::chrono::duration_cast<std::chrono::duration<double>>(ns)).count();
   }
 
+  inline double timeToSec(const rclcpp::Time & rclcpp_time)
```

Comment:
This would probably be better suited to be a method on the `rclcpp::Time` class similar to https://github.com/ros2/rclcpp/blob/master/rclcpp/include/rclcpp/time.hpp#L103 . Or at least not exposed publicly here.

Reply:
Added `rclcpp::Time::seconds()` in ros2/rclcpp#526

Reply:
866ab004e6500c48b76903d2c7ffed7441eaf741